backend/coins/eth/account.go

```diff
@@ -1028,7 +992,75 @@ func (account *Account) MatchesAddress(address string) (bool, error) {
 // EnqueueUpdate enqueues an update for the account.
 func (account *Account) EnqueueUpdate() {
 	select {
-	case account.enqueueUpdateCh <- struct{}{}:
+	case account.enqueueUpdateCh <- account:
 	default:
```

Comment:
Now that the same channel handles multiple accounts, you sholud remove the default statement. Otherwise, if one account is enqueued to be updated, but not processed yet, further ones will be ignored. I suspect that if you load many ETH accounts like this, not all of them will actually be initialized.